modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -694,7 +712,7 @@ performance, this value should not be greater than the number of available proce
 |===
 
 [[import-tool-incremental-examples]]
-=== Examples
+=== Two ways to import data incrementally / Two modes of the incremental import
```

Comment:
All tool's sections follow the same/similar structure - short introductory sentences, syntax, description, usage, and examples. The idea is that if users know one tool, it will be easier for them to find information for another one. So, I would suggest that we keep the structure as it is. Also, we'll need to change the headings every time we add another example.

Reply:
Yes, I see that tool's sections are structured the same way and I agree that it makes sense. On the other side, I don't think that the title 'Examples' is informative enough in this particular case. Actually, we're not talking about the examples on how to apply `neo4j-admin` syntax to import data incrementally , but we're talking about two different ways to run the incremental import. I'm suggesting to think on alternatives. If the 'Examples' is the best way, let's keep it.